README.MD

```diff
@@ -2,12 +2,13 @@
 During workshop you will be developing weather-station and weather-forecast services that provides information about
 weather conditions. To be able to complete tasks described in workshop you will need:
 
-1. JDK8 (recommended) or newer
+1. JDK8 or newer
 2. IDE
 3. Internet connection
 4. Browser
 
-**Note**: Workshop was created and tested using Intellij Community Edition
+**Note**: Workshop was created and tested using Intellij Community Edition with Lombok plugin and enabled annotation processing.
```

Comment:
I've removed dependency on lombok few commits before to avoid plugin installation and additional steps. Information about it is no longer needed.